test/tests/performance/pbs_cgroups_stress.py

```diff
@@ -199,14 +199,26 @@ def test_cgroups_epilogue_race(self):
         self.server.manager(MGR_CMD_SET, SERVER, attr)
         self.load_config(self.cfg0)
         now = time.time()
-        j = Job(TEST_USER, attrs={ATTR_J: '1-1000'})
+        j = Job(TEST_USER, attrs={ATTR_J: '0-1000'})
         j.create_script(self.true_script)
         jid = self.server.submit(j)
         jid = jid.split('[')
-        for i in range(1, 1001):
+        done = False
+        for i in range(0, 1000):
+            sjid = jid[0] + "[" + str(i) + "]"
+            if done:
+                self.server.expect(
+                    JOB, {'job_state': 'F'}, id=sjid, extend='tx')
```

Comment:
Don't need to 't' here.  Just need the 'x'

Reply:
Removed the t here and below